.drone.jsonnet

```diff
@@ -490,6 +491,16 @@ local integration_cilium_strict = Step('e2e-cilium-strict', target='e2e-qemu', p
   WITH_CONFIG_PATCH: '[{"op": "add", "path": "/cluster/network", "value": {"cni": {"name": "none"}}}, {"op": "add", "path": "/cluster/proxy", "value": {"disabled": true}}]',
```

Comment:
I think I somehow removed `WITH_FIREWALL: 'accept'` from this block

Reply:
yes, or could be my rebase, fixed, anyways it should pass, worked locally, running again